windowsforms/TaskDialogDemo/vb/README.md

```diff
@@ -0,0 +1,31 @@
+---
+languages:
+- vb
+products:
+- dotnet
+- windows
+page_type: sample
+name: "Windows Forms Sample: Task Dialog (Visual Basic)"
+urlFragment: "winforms-taskdialog-vb"
+description: ".NET 5.0 Windows Forms application that demonstrates how to use Task Dialog in Visual Basic."
+---
+
+# Task Dialog Demo
+
+This sample is a .NET 5.0 Windows Forms Visual Basic application that demonstrates how to use [`Task Dialog`](https://docs.microsoft.com/en-us/windows/win32/controls/task-dialogs-overview).
+
+![Screenshot](../images/screenshot.jpg)
+
+Credits: this sample was authored by @kpreisser, the author of [`Task Dialog` for Windows Forms](https://github.com/dotnet/winforms/pull/1133).
+
+## Sample prerequisites
+
+This sample is written in Visual Basic and targets .NET 5.0 running on Windows. It requires the [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0).
+
+## Building the sample
+
+Build and run the demo:
+
+```
+TaskDialogDemo\vb> dotnet run
+```
```

Comment:
Changing "Building" section to be similar to what is used by the other examples.

```suggestion
## Building the sample

To download and run the sample, follow these steps:

01. Download and unzip the sample.

01. In Visual Studio 2019 or later:

    01. On the menu bar, choose **File** > **Open** > **Project/Solution**.

    01. Navigate to the folder that holds the unzipped sample code, and open the project (*.vbproj*) file.

    01. Choose the F5 key, or Ctrl+F5 keys, to run the project.

01. From the command line:

    01. Navigate to the folder that holds the unzipped sample code.

    01. At the command line, type `dotnet run`.
    
```